lib/corpus.py

```diff
@@ -459,6 +462,9 @@ def dump(self, out: TextIO, indentation: str = ""):
         else:
             out.write("</segment>\n")
 
+    def __repr__(self):
+        return f"{self.__class__.__name__}({self.fullname()}:{self.start:.02f}-{self.end:.02f})"
```

Comment:
```suggestion
        return f"<{self.__class__.__name__} {self.fullname()}:{self.start:.02f}-{self.end:.02f}>"
```

Reply:
Is the colon character disallowed in segment names? Otherwise or even then the notation might lead to confusion as to what is part of the name and what are the timestamps.

```suggestion
        return f"<{self.__class__.__name__} {self.fullname()} {self.start:.02f}-{self.end:.02f})>"
```
or maybe even 
```suggestion
        return f"<{self.__class__.__name__} {self.fullname()} start={self.start:.02f} end={self.end:.02f})>"
```

Reply:
Thanks for the suggestions. I applied the one with spaces since I like that most. Feel free to comment if you disagree.